app/commands/solution/mark_as_out_of_date_in_index.rb

```diff
@@ -0,0 +1,32 @@
+class Solution::MarkAsOutOfDateInIndex
```

Comment:
I wonder if this should be `Exercism::MarkSolutionsAsOutOfDateInIndex` as it's acting on an exercise, not a solution. I could see it either way. Thoughts?

Reply:
Yeah, I considered that. I'm perfectly happy with that.